src/main/java/bio/terra/cli/command/resources/create/AiNotebook.java

```diff
@@ -71,7 +71,7 @@
           "https://raw.githubusercontent.com/DataBiosphere/terra-cli/main/notebooks/post-startup.sh",
       description =
           "Path to a Bash script that automatically runs after a notebook instance fully boots up. "
-              + "The path must be a URL or Cloud Storage path, e.g. 'gs://path-to-file/file-name'")
+              + "The path must be a URL or Cloud Storage path, e.g. 'gs://path-to-file/file-name'.")
```

Comment:
URI I think they call it

Reply:
cloud storage [documentation](https://cloud.google.com/storage/docs/cloud-console#_accessing) calls it URL